.circleci/config.yml

```diff
@@ -218,6 +218,9 @@ jobs:
       - run:
           name: "Test: Build and Spin-up Django API Service"
           command: "cd tdrs-backend; docker-compose up -d --build"
+      - run:
+          name:  "Test: Execute Python Linting Test"
```

Comment:
This step was moved to before unit tests are run to allow some extra time for the localstack-setup.sh script to run. It isn't run until localstack is deemed "ready" so it can happen after the `wait-for-services.sh` script has allowed the backend to start up.

By running the linter step first we can ensure that script has time to run and also developers get the added benefit of seeing linter failures earlier in the pipeline - which can help reduce some time churn that is spent on getting CI passing since you have to wait for all the tests to pass to currently see linter errors in CI.